pkg/entrypoint/entrypointer.go

```diff
@@ -91,6 +91,14 @@ func (e Entrypointer) Go() error {
 			// An error happened while waiting, so we bail
 			// *but* we write postfile to make next steps bail too.
 			e.WritePostFile(e.PostFile, err)
+			output = append(output, v1alpha1.PipelineResourceResult{
+				Key:   "StartedAt",
+				Value: time.Now().Format(time.RFC3339),
+			})
+
+			if wErr := termination.WriteMessage(e.TerminationPath, output); wErr != nil {
```

Comment:
could we do this once in a `defer`-ed func instead of duplicating it?

Reply:
Done, thanks